src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -1050,25 +1060,10 @@ class AlterTableStatementSegment(BaseSegment):
                         optional=True,
                     ),
                 ),
-                # Add index
+                # Add constraint
                 Sequence(
                     "ADD",
-                    Ref.keyword("UNIQUE", optional=True),
-                    OneOf("INDEX", "KEY", optional=True),
-                    Ref("IndexReferenceSegment"),
-                    Ref("IndexTypeGrammar", optional=True),
-                    Ref("BracketedColumnReferenceListGrammar"),
-                    AnySetOf(
-                        Sequence(
-                            "KEY_BLOCK_SIZE",
```

Comment:
Why was this deleted? I don't see similar code in TableConstraintSegment.

Reply:
With these changes, is it still possible to use `KEY_BLOCK_SIZE`? (Strangely, there is no `.sql` file using this.) Could you please add a test case for it to your PR?

You could use one of the examples from this page: https://dev.mysql.com/doc/refman/5.7/en/innodb-compression-usage.html

Reply:
> With these changes, is it still possible to use KEY_BLOCK_SIZE? (Strangely, there is no .sql file using this.)

Yes.

> Could you please add a test case for it to your PR?

Added in 251294722ed48b2979577b2250870b1499051e9f
Thanks

Reply:
Thanks!